examples/gaussian_processes/HSGP-Advanced.myst.md

```diff
@@ -0,0 +1,896 @@
+axs["True"].text(
+    train_text_x,
+    text_y,
+    "Training territory",
+    horizontalalignment="center",
+    verticalalignment="center",
+    fontsize=14,
+    color="blue",
+    alpha=0.7,
+)
+axs["True"].text(
+    test_text_x,
+    text_y,
+    "Testing territory",
+    horizontalalignment="center",
+    verticalalignment="center",
+    fontsize=14,
+    color="green",
+    alpha=0.7,
+)
+
+axs["Data"].axvline(x_train[-1], ls=":", lw=3, color="k", alpha=0.6)
+axs["Preds"].axvline(x_train[-1], ls=":", lw=3, color="k", alpha=0.6)
+
+# Plot mean GP
+axs["Preds"].fill_between(
+    x_full,
+    pred_f_mu_mu - pred_f_mu_sd,
+    pred_f_mu_mu + pred_f_mu_sd,
+    color="C1",
+    alpha=0.8,
+    edgecolor="none",
+)
+
+for i in range(n_gps):
+    axs["True"].plot(x_train, f_true_train[:, i], color=colors_train[i])
+    axs["True"].plot(x_test, f_true_test[:, i], color=colors_test[i])
+    axs["Data"].scatter(x_train, y_train[:, i], color=colors_train[i], alpha=0.6)
+    axs["Data"].scatter(x_test, y_test[:, i], color=colors_test[i], alpha=0.6)
+    # Plot inferred GPs with uncertainty
+    axs["Preds"].fill_between(
+        x_train,
+        pred_f_mu[:n_train, i] - pred_f_sd[:n_train, i],
+        pred_f_mu[:n_train, i] + pred_f_sd[:n_train, i],
+        color=colors_train[i],
+        alpha=0.3,
+        edgecolor="none",
+    )
+    axs["Preds"].fill_between(
+        x_test,
+        pred_f_mu[n_train:, i] - pred_f_sd[n_train:, i],
+        pred_f_mu[n_train:, i] + pred_f_sd[n_train:, i],
+        color=colors_test[i],
+        alpha=0.3,
+        edgecolor="none",
+    )
+
+axs["True"].set(xlabel="x", ylim=ylims, title="True GPs\nMean GP in orange")
+axs["Data"].set(xlabel="x", ylim=ylims, title="Observed data\nColor corresponding to GP")
+axs["Preds"].set(
+    xlabel="x",
+    ylim=ylims,
+    title="Predicted GPs, $\\pm 1 \\sigma$ posterior intervals\nMean GP in orange",
+);
+```
+
+# Example 2: An HSGP that exploits Kronecker structure
+
++++ {"editable": true, "slideshow": {"slide_type": ""}}
+
+This example is a multiple GP model like the previous one, but it assumes a different relationship between the GPs. **Instead of pooling towards a common mean GP, there is an additional covariance structure that specifies their relationship**.
+
+For example, we may have time series measurements of temperature from multiple weather stations. The similarity over time should mostly depend only on the _distance_ between the weather stations. They all will likely have the same dynamics, or same covariance structure, over time. You can think of this as _local_ partial pooling.  
+
+In the example below, we arrange the GPs along a single "spatial" axis, so it's a 1D problem and not 2D, and then allow them to share the same time covariance.  This might be clearer after taking a look at the simulated data below.  
+
+Mathematically, this model uses the [Kronecker product](GP-Kron.myst.md), where the "space" and "time" dimensions are _separable_.
```

Comment:
same here: https://pymcio--668.org.readthedocs.build/projects/examples/en/668/gaussian_processes/HSGP-Advanced.html#example-2-an-hsgp-that-exploits-kronecker-structure no cross-reference being generated

Reply:
Done